rotatingtext/src/main/java/com/sdsmdg/harjot/rotatingtext/models/Rotatable.java

```diff
@@ -48,50 +48,32 @@ public Rotatable(int updateDuration, String... text) {
     }
 
     public Rotatable(int color, int updateDuration, String... text) {
-        colorList.add(color);
+        this.color_array = new Integer[]{color};
```

Comment:
Is it absolutely necessary to have this line here? Remove it from this rotatable if possible.

Reply:
I have created a new function useArray() and removed the statement in rotatable .